lib/grinder.dart

```diff
@@ -317,10 +317,9 @@ class Grinder {
     if (targets.isEmpty) {
       if (defaultTask != null) {
         targets = [defaultTask.name];
-      } else {
-        if (!dontRun) log('no tasks specified, and no default task defined');
+      } else if (!dontRun) {
+        throw new GrinderException('Tried to call non-existent default task.');
```

Comment:
It looks like this change will mean that running grinder w/o a task specified will print `Tried to call non-existent default task` instead of `run 'grinder -h' for help and a list of valid tasks`. I think we're better off w/ this not being an error condition -